administration/src/project-configs/bayern/config.ts

```diff
@@ -83,6 +83,7 @@ const config: ProjectConfig = {
       primaryColorLight: '#c6c0d8',
     },
   },
+  cardCreation: true,
```

Comment:
nit-picky one:
maybe to follow the style we already have
```suggestion
cardCreation: {
  enabled: false,
},
```

Reply:
i think we just need this object style if we add some additional parameters/configs if it is enabled. Which we don't do here, so i think thats not needed here